### PR TITLE
ref: Add autoformatting for JSON

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

**Comment by untitaker** [comment]
as noted on https://github.com/getsentry/action-github-commit -- need to use a different GITHUB_TOKEN. probably using https://github.com/getsentry/action-github-app-token

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,6 +7,33 @@ on:
...
+          private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - run: make install format
+
+      - uses: getsentry/action-github-commit@bec0adb2924f8c19a9f84c18c83d375caf02ca38 # main
+        if: github.ref != 'refs/heads/master'
```
Comment (lynnagara) [inline .github/workflows/ci.yml:32]: does it need to be main?

Comment (untitaker) [reply .github/workflows/ci.yml:32]: you are right! fixed

File: Makefile [hunk 2]
```diff
@@ -23,6 +24,8 @@ install: python/sentry_kafka_schemas/schema_types
 
 format:
 	black python/
+	cargo fmt
+	yarn prettier --write .
```
Comment (lynnagara) [inline Makefile:28]: Is it possible to do something like:
```suggestion
	yarn prettier --write **.*.json
```

**Review by lynnagara — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (untitaker) [reply Makefile:28]: @lynnagara yeah I would say whatever prettier supports is fair game. It doesn't seem to touch files it doesn't understand. Unlikely it conflicts with black :D

**Pull request merged**